packages/drivers/odsp-driver/src/odspDeltaStorageService.ts

```diff
@@ -49,62 +50,70 @@ export class OdspDeltaStorageService {
 			const baseUrl = this.buildUrl(from, to);
 			const storageToken = await this.getStorageToken(options, "DeltaStorage");
 
-			const formBoundary = uuid();
-			let postBody = `--${formBoundary}\r\n`;
-			postBody += `Authorization: Bearer ${storageToken}\r\n`;
-			postBody += `X-HTTP-Method-Override: GET\r\n`;
-
-			postBody += `_post: 1\r\n`;
-			postBody += `\r\n--${formBoundary}--`;
-			const headers: { [index: string]: any } = {
-				"Content-Type": `multipart/form-data;boundary=${formBoundary}`,
-			};
-
-			// Some request take a long time (1-2 minutes) to complete, where telemetry shows very small amount
-			// of time spent on server, and usually small payload sizes. I.e. all the time is spent somewhere in
-			// networking. Even bigger problem - a lot of requests timeout (based on cursory look - after 1-2 minutes)
-			// So adding some timeout to ensure we retry again in hope of faster success.
-			// Please see https://github.com/microsoft/FluidFramework/issues/6997 for details.
-			const abort = new AbortController();
-			const timer = setTimeout(() => abort.abort(), 30000);
-
-			const response = await this.epochTracker.fetchAndParseAsJSON<IDeltaStorageGetResponse>(
-				baseUrl,
+			return PerformanceEvent.timedExecAsync(
+				this.logger,
 				{
-					headers,
-					body: postBody,
-					method: "POST",
-					signal: abort.signal,
+					eventName: "OpsFetch",
+					attempts: options.refresh ? 2 : 1,
+					from,
+					to,
+					...telemetryProps,
+					reason: scenarioName,
+				},
+				async (event) => {
+					const formBoundary = uuid();
```

Comment:
(nit) you can probably move constant thigs up, above performance event, as there is no need to calculate them twice.

Reply:
We are only calculating them twice in case we do token fetch call twice, which we will still do even if I move it above the perf event. So it does not make any difference.